libcxx/test/support/msvc_stdlib_force_include.h

```diff
@@ -103,5 +103,6 @@ const AssertionDialogAvoider assertion_dialog_avoider{};
 #define TEST_ABI_MICROSOFT
 
 #define _LIBCPP_AVAILABILITY_THROW_BAD_ANY_CAST
+#define _LIBCPP_AVAILABILITY_HAS_FROM_CHARS_FLOATING_POINT 1
```

Comment:
Not necessary if you take my suggestion in `test_macros.h`, cause we'll take for granted that fp `from_chars` is available on non-libc++.

Reply:
I think this line can be removed now, since `test_macros.h` will assume `TEST_HAS_FROM_CHARS_FLOATING_POINT` when running against MSVC stdlib.